pennylane/ops/functions/__init__.py

```diff
@@ -45,3 +45,4 @@
 from .map_wires import map_wires
 from .matrix import matrix
 from .simplify import simplify
+from .iterative_qpe import iterative_qpe

```

Comment:
Add it to the doc-string above in this file

Reply:
oh good catch